lib/cloudant.js

```diff
@@ -948,7 +942,7 @@ Cloudant.prototype.updateIndex = function(mo, modelName, cb) {
     },
   };
   /* eslint-enable camelcase */
-  debug('Cloudant.prototype.updateIndex modelName %j', modelName);
+  debug('Cloudant.prototype.updateIndex modelName %j idx %j', modelName, idx);
```

Comment:
`debug('Cloudant.prototype.updateIndex -- modelName: %s, idx: %s', modelName, idx)`

If the modelName or idx are objects and not strings for some weird reason, you can use `require('util').inspect` to stringify them nicely.